### ISSUE TITLE
gopanel - Web interface for multi-log goaccess realtime

### ISSUE THREAD

**Issue by neocogent** [allinurl/goaccess#1329]
I couldn't find a place to announce this so decide to try here. Maybe it would be worth starting a reddit for goaccess so users could communicate?

Anyway, I've coded up [gopanel](https://github.com/neocogent/gopanel) - a simple python (flask) web interface for goaccess allowing a web admin to view multiple logs in realtime easily with a menu interface. I'm wondering if others find this useful and if so I may extend it with better setup, settings and summary stats on the menu items. Settings could allow for adding global or per log filters and managing more form the interface rather than conf files.

I have it working on a couple of my servers with about 6 sites, eah having a log file menu entry. That's not extensive testing but so far it's working ok.

Welcome any feedback.

**Comment by allinurl** [comment]
I agree, so far I think it would be useful to have a reddit for goaccess, but let me think about this a bit more before opening one.

Thanks a lot for sharing [**gopanel**](https://github.com/neocogent/gopanel), it sounds great! I'll give it a shot and post back my feedback. Would you be able to post a screenshot in the project page and I can share it in twitter as I think it would very helpful.

**Comment by neocogent** [comment]
Will do that. It's pretty plain vanilla right now. If people like the idea I can spiff it up a bit. For my needs the vanilla look is ok. It's a start. There are many improvements possible.
